modules/control-utility/src/main/java/org/apache/ignite/internal/commandline/systemview/SystemViewCommand.java

```diff
@@ -0,0 +1,223 @@
+import org.apache.ignite.internal.visor.systemview.VisorSystemViewTask;
+import org.apache.ignite.internal.visor.systemview.VisorSystemViewTask.SimpleAttributeType;
+import org.apache.ignite.internal.visor.systemview.VisorSystemViewTaskArg;
+import org.apache.ignite.internal.visor.systemview.VisorSystemViewTaskResult;
+import org.apache.ignite.spi.systemview.view.SystemView;
+
+import static java.util.Collections.nCopies;
+import static org.apache.ignite.internal.commandline.CommandList.SYSTEM_VIEW;
+import static org.apache.ignite.internal.commandline.CommandLogger.optional;
+import static org.apache.ignite.internal.commandline.TaskExecutor.executeTaskByNameOnNode;
+import static org.apache.ignite.internal.commandline.systemview.SystemViewCommandArg.NODE_ID;
+import static org.apache.ignite.internal.visor.systemview.VisorSystemViewTask.SimpleAttributeType.DATE;
+import static org.apache.ignite.internal.visor.systemview.VisorSystemViewTask.SimpleAttributeType.NUMBER;
+import static org.apache.ignite.internal.visor.systemview.VisorSystemViewTask.SimpleAttributeType.STRING;
+
+/**
+ * Represents command for {@link SystemView} content printing.
+ */
+public class SystemViewCommand implements Command<Object> {
+    /** Column separator. */
+    public static final String COLUMN_SEPARATOR = "    ";
+
+    /**
+     * Argument for the system view content obtainig task.
+     *
+     * @see VisorSystemViewTask
+     */
+    private VisorSystemViewTaskArg taskArg;
+
+    /** ID of the node to get the system view content from. */
+    private UUID nodeId;
+
+    /** {@inheritDoc} */
+    @Override public Object execute(GridClientConfiguration clientCfg, Logger log) throws Exception {
+        try {
+            VisorSystemViewTaskResult res;
+
+            try (GridClient client = Command.startClient(clientCfg)) {
+                res = executeTaskByNameOnNode(
+                    client,
+                    VisorSystemViewTask.class.getName(),
+                    taskArg,
+                    nodeId,
+                    clientCfg
+                );
+            }
+
+            if (res != null)
+                printSystemViewContent(res, log);
+            else
+                log.info("No system view with specified name was found [name=" + taskArg.systemViewName() + "]");
+
+            return res;
+        }
+        catch (Throwable e) {
+            log.severe("Failed to perform operation.");
+            log.severe(CommandLogger.errorMessage(e));
+
+            throw e;
+        }
+    }
+
+    /**
+     * Prints system view content obtained via {@link VisorSystemViewTask} execution.
+     *
+     * @param taskRes Result of {@link VisorSystemViewTask} execution.
+     * @param log Logger.
+     */
+    private void printSystemViewContent(VisorSystemViewTaskResult taskRes, Logger log) {
+        List<String> colTitles = taskRes.systemViewAttributes();
+
+        List<? extends List<?>> sysViewRows = taskRes.systemViewContent();
```

Comment:
? extends List<?> can be replaced with List<?>

Reply:
Done.